.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
if you add this to your `~/.gitignore` you never have to worry about seeing it in your repos 😉